docs/requirements.txt

```diff
@@ -0,0 +1,21 @@
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+-e .[all]
```

Comment:
Since we don't have `setup.py` we can remove `-e .[all]`and we can add documentation requirements here, `Sphinx>=1.7.5` and `sphinx-rtd-theme >=0.4.0`. Also, in `run-tests.sh` we can build the docs, see for example [workflow-controller](https://github.com/reanahub/reana-workflow-controller/blob/master/run-tests.sh), so it would fail if there is any error in the docs. You could also plug Travis CI, see [workflow-controller .travis.yml](https://github.com/reanahub/reana-workflow-controller/blob/master/.travis.yml) (things like docker and sudo true are not needed for this package, and `pip install -e .[all]` becomes `pip install -r requirements.txt`).